custom_endpoint/src/Controller/CustomEndpointController.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Drupal\custom_endpoint\Controller;
+
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Drupal\Core\Session\AccountProxy;
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Core\Session\AccountInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drupal\custom_endpoint\CurrentUserService;
+
+/**
+ * Custom Endpoint Controller class
+ */
+class CustomEndpointController extends ControllerBase {
+
+  public $currentUserService;
+
+  /**
+   * Custom Endpoint controller constructor
+   *
+   * @param Drupal\custom_endpoint\CurrentUserService
+   */
+  public function __construct(CurrentUserService $currentUser) {
```

Comment:
parameters and variables in Drupal are snake-cased, so in this case:
`$current_user`

only class members are camel-cased

Reply:
fixed